docs/DeveloperGuide.md

```diff
@@ -415,6 +415,17 @@ _{Explain here how the data archiving feature will be implemented}_
 - Modify the pre-defined set of allowed characters to **capture any character instead**. This allows for flexibility since all special characters will be accepted for the `ADDRESS` field.
 - For addresses containing `/` characters, the user will be required to use the `\` character to escape each `/` character similar to the example in Planned Enhancement 1.
 
+### 3. Add message notification for data file loading process
```

Comment:
Do we still require this @Javiery3889 @Dethada @Arixeyeion if we are no longer planning to support editing of JSON file outside of FINDvisor?

Reply:
This one is for loading of data right? Like I save my contacts in the JSON then just copy over to a new computer?

Reply:
I think we should have it, useful in the case where the data file is corrupted for some reason, might not be due to manual modification.

Reply:
@KohGuanZeh @YuZizhen @Arixeyeion I agreww with what @Dethada mentioned, would be better if we include it